### PR TITLE
Accept amount suffixed with "ae" instead of "denomination" option

### PR DESCRIPTION
**Pull request by davidyuk**

### CODE REVIEW

File: test/account.js [hunk 1]
```diff
@@ -69,26 +69,25 @@ describe('Account Module', () => {
...
+      'spend', WALLET_NAME, '--password', 'test', '--json',
+      receiverKeys.publicKey, '1ae', '--fee', '0.02ae',
+    ]);
+    expect(await sdk.getBalance(receiverKeys.publicKey)).to.be.equal('1000000000000000000');
+    expect(fee).to.be.equal('20000000000000000');
   });
 
   it('Spend fraction of coins to account by name', async () => {
-    const fraction = 0.0001;
+    const fraction = 0.000001;
```
Comment (davidyuk) [inline test/account.js:90]: Because of inaccurate number comparison from below

File: test/account.js [hunk 2]
```diff
@@ -69,26 +69,25 @@ describe('Account Module', () => {
   });
 
   it('Spend coins to another wallet', async () => {
-    const amount = 100;
+    const amount = '100';
```
Comment (davidyuk) [inline test/account.js:72]: In normal usage cli arguments are always strings 🤔

File: test/index.js [hunk 3]
```diff
@@ -87,7 +87,7 @@ export async function executeProgram(program, args) {
   const options = getProgramOptions(program);
   try {
     const allArgs = [
-      ...args,
+      ...args.map((arg) => arg.toString()),
```
Comment (davidyuk) [inline test/index.js:90]: In normal usage cli arguments always strings 🤔

**Review by thepiwo — APPROVED** [review]
lgtm

**Pull request merged**